### ISSUE TITLE
pip install thop failed

### ISSUE THREAD

**Issue by InstantWindy** [Lyken17/pytorch-OpCounter#5]
Hi! I want to install this file offline through pip install thop. But it appear errors.
Environment:
   unbuntu 
  python 3.6
  torch 0.4.1
![image](https://user-images.githubusercontent.com/23608180/50090681-4c247480-0244-11e9-8675-3c3c8419e75b.png)
Thanks!

**Comment by Lyken17** [comment]
Looks like a network issue. Can you try to install it again?

**Comment by InstantWindy** [comment]
I try it again,but it still appears same error.

**Comment by Lyken17** [comment]
Where is your physical location? If mainland China, it is very likely that pip suffers from packet loss. Have you tried any proxy?

**Comment by Lyken17** [comment]
In any case, if pip continues to fail, you can clone the project and install by `python setup.py install`.

**Comment by InstantWindy** [comment]
Yeah ,it works .thanks!Is InstanceNorm2d the same as BatchNorm2d? Is AdaptiveAvgPool2d the same as AvgPool2d? Is Sigmoid the same as Softmax?

**Comment by Lyken17** [comment]
`InstanceNorm2d` is not exactly the same as `BatchNorm2d`, I will add one later.

`AvgPool2d` with proper parameters is the same as `AdaptiveAvgPool2d`. 

Please refer to PyTorch documentation for the layer details.
